index.html

```diff
@@ -15,22 +15,22 @@
                 </nav>
             </header>
             <div class = "container">
```

Comment:
- [ ] This second part of the page should have their container as a `section` tag.
    - The container of the section is a section tag.
       - It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

- [ ] Also the social media icon should not be wrapped inside of a `section` tag, rather it should be wrapped inside of a `ul` with an `li` and each `li` with an `a`
   - The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).
      - Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.

E.g :point_down: 
```HTML
<ul>
  <li><a href="#">social media icon</a></li>
  <li><a href="#">social media icon</a></li>
  <li><a href="#">social media icon</a></li>
  <li><a href="#">social media icon</a></li>
</ul>
```